zipline/pipeline/term.py

```diff
@@ -436,7 +439,12 @@ def _validate(self):
         if self.window_length:
             for child in self.inputs:
                 if child.windowed:
```

Comment:
I don't think you want the `.windowed` check anymore.  You just want `if not child.window_safe: error()`.